static/skywire-manager-src/src/assets/i18n/de.json

```diff
@@ -237,48 +296,116 @@
-      "autostart-enabled": "Autostart ein"
+      "autostart-enabled": "Autostart ein",
+      "unavailable-logs-error": "Kann Logs nicht zeigen, solange die Anwendung gestoppt ist.",
+    
+      "filter-dialog": {
+        "state": "Der Status muss sein",
+        "name": "Der Name muss enthalten",
+        "port": "Der Port muss enthalten",
+        "autostart": "Autostart muss sein",
+  
+        "state-options": {
+          "any": "Läuft oder gestoppt",
+          "running": "Läuft",
+          "stopped": "Gestoppt"
+        },
+
+        "autostart-options": {
+          "any": "An oder Aus",
+          "enabled": "An",
+          "disabled": "Aus"
+        }
+      }
     },
-    "skysocks-settings": {
-      "title": "Skysocks Einstellungen",
+    "vpn-socks-server-settings": {
+      "socks-title": "Skysocks Einstellungen",
+      "vpn-title": "VPN-Server Einstellungen",
       "new-password": "Neues Passwort (Um Passwort zu entfernen leer lassen)",
       "repeat-password": "Passwort wiederholen",
       "passwords-not-match": "Passwörter stimmen nicht überein.",
+      "secure-mode-check": "Sicherheitsmodus benutzen",
+      "secure-mode-info": "Wenn aktiv, erlaubt der Server kein Client/Server SSH und erlaubt kein Datenverkehr vom VPN-Client zum lokalen Netzwerk des Servers.",
       "save": "Speichern",
       "remove-passowrd-confirmation": "Kein Passwort eingegeben. Wirklich Passwort entfernen?",
       "change-passowrd-confirmation": "Passwort wirklich ändern?",
       "changes-made": "Änderungen wurden gespeichert."
     },
-    "skysocks-client-settings": {
-      "title": "Skysocks-Client Einstellungen",
-      "remote-visor-tab": "Remote Visor",
+    "vpn-socks-client-settings": {
+      "socks-title": "Skysocks-Client Einstellungen",
+      "vpn-title": "VPN-Client Einstellungen",
+      "discovery-tab": "Suche",
+      "remote-visor-tab": "Manuelle Eingabe",
       "history-tab": "Verlauf",
+      "settings-tab": "Einstellungen",
+      "use": "Diese Daten benutzen",
+      "change-note": "Notiz ändern",
+      "remove-entry": "Eintrag löschen",
+      "note": "Notiz:",
+      "note-entered-manually": "Manuell eingegeben",
+      "note-obtained": "Von Discovery-Service erhalten",
+      "key": "Schlüssel:",
+      "port": "Port:",
+      "location": "Ort:",
+      "state-available": "Verfügbar",
+      "state-offline": "Offline",
       "public-key": "Remote Visor öffentlicher Schlüssel",
+      "password": "Passwort",
+      "password-history-warning": "Achtung: Das Passwort wird nicht im Verlauf gespeichert.",
+      "copy-pk-info": "Öffentlichen Schlüssel kopieren.",
+      "copied-pk-info": "Öffentlicher Schlüssel wurde kopiert",
+      "copy-pk-error": "Beim Kopieren des öffentlichen Schlüssels ist ein Problem aufgetreten.",
+      "no-elements": "Derzeit können keine Elemente angezeigt werden. Bitte später versuchen.",
+      "no-elements-for-filters": "Keine Elemente, welche die Filterkriterien erfüllen",
```

Comment:
May need punctuation. Not sure what would be correct since this is not really a sentence, but feels like one. 😄

Reply:
Yes, maybe also `Keine Elemente vorhanden, ...` ?